setup.py

```diff
@@ -68,6 +68,7 @@ def read(path):
               'zope.testrunner>=5,<7',
               'zc.customdoctests>=1.0.1,<2',
               'createcoverage>=1,<2',
+              'dask',
```

Comment:
I am not sure but shouldn't we have here `dask[complete]` to install all dask dependencies (e.g., `dask.dataframe`)

Reply:
This list is only used for testing the functionality, we do not install `pandas` or `dask` together with the library. As such, I think it is fine to use a minimal set of dependencies, which is enough to satisfy the requirements of the test suite.